config/meteor.yaml.sample

```
@@ -0,0 +1,7 @@
+LOG_LEVEL: hehehehehe
```

we can make the default to `info` or `debug` 😄 
```suggestion
LOG_LEVEL: debug
```